### DISCUSSION TITLE
I'm very new to this and am having trouble getting my server to start.

### DISCUSSION THREAD

**Discussion by CaedmonB05** [itzg/docker-minecraft-server#2438] [Q&A]
[minecraft docker.txt](https://github.com/itzg/docker-minecraft-server/files/12911695/minecraft.docker.txt)
the top is my mods.txt, and the bottom is the docker-compose.yml. 
It has an error while starting and loops because of the restart command
I think multiple errors are coming from the same issue, but I'm unsure.
[errors.txt](https://github.com/itzg/docker-minecraft-server/files/12911829/errors.txt)
sorry if this has repeating issues, I copied everything I thought was new information

**Comment by itzg** [comment]
There seems to be an issue with one of the mods, but that's not an issue specific to this docker image. You'll need to troubleshoot which mod is at fault by trial and error removing/adding them and/or looking at the surrounding the mods. You may also want to contact people on a FabricMC discussion forum.
